static/src/javascripts/projects/common/modules/commercial/hosted-about.js

```diff
@@ -0,0 +1,31 @@
+define([
+    'bean',
+    'common/utils/$',
+    'common/modules/commercial/survey/survey-simple'
+], function (
+    bean,
+    $,
+    SurveySimple
+) {
+    function init() {
+        var survey = new SurveySimple({
+            id: 'hosted-about',
+            header: 'Advertiser content',
+            paragraph1: 'Advertiser content is used to describe advertisement features that are paid for, produced and controlled by the advertiser rather than the publisher​.',
+            paragraph2: 'They​ are subject to regulation by the Advertising Standards Authority in the UK, the Federal Trade Commission in the US and the Advertising Standards Bureau in Australia.',
+            paragraph3: 'This content is produced by the advertiser and does not involve GNM staff.',
+            showCloseBtn: true
+        });
+
+        survey.attach();
+
+        bean.on(document, 'click', $('.js-hosted-about'), function (e) {
```

Comment:
There will be at least 2 `.js-hosted-about` on the page and those elements don't have any common parent. There is a lot `bean.on(document ...` examples in our codebase. If I understood the bean documentation correctly, there can be only one element passed to the `bean.on` method (without delegation). I assume that using the `bean.on(document ...` method is the best way to attach the same event to multiple objects.